core/src/main/scripts/importer/validateData.py

```diff
@@ -2898,3 +2898,4 @@ def main_validate(args):
                 1: 'failed',
                 2: 'not performed as problems occurred',
                 3: 'succeeded with warnings'}.get(exit_status, 'unknown')))
+    sys.exit(exit_status)
```

Comment:
the metaImport calls the validateData, so in this case we would like to *not* have a sys.exit (especially in case of status == 0). Maybe only do this if status != 0 ?